File: packages/coding-agent/test/suite/explore/fixtures/ts/order-service.ts
/** Service that processes orders against inventory. */
export class OrderService {
	private readonly taxRate: number;
	private pending: number;

	constructor(taxRate: number) {
		this.taxRate = taxRate;
		this.pending = 0;
	}

	/** Compute the total for an order including tax. */
	computeTotal(amount: number): number {
		return amount + amount * this.taxRate;
	}

	reset(): void {
		this.pending = 0;
	}
}

File: packages/coding-agent/test/suite/explore/fixtures/py/order_service.py
class OrderService:
    """Service that processes orders against inventory."""

    def __init__(self, tax_rate: float) -> None:
        self.tax_rate = tax_rate
        self.pending = 0

    def compute_total(self, amount: float) -> float:
        """Compute the total for an order including tax."""
        return amount + amount * self.tax_rate

    def reset(self) -> None:
        self.pending = 0
